### PR TITLE
5.4.0 updates fixes

### PR DESCRIPTION
**Pull request by agarstang**
Update to latest images and packages.
Fix Caddy URL.
Fix various linting and style warnings.

**WARNING:** puppetexplorer now downloads caddy with the personal license as is required (perhaps remove this or add license warning to README).

### CODE REVIEW

**Comment by agarstang** [comment]
This might be helpful for @shrug. This is what I needed to do to get 5.4.0 working.

**Comment by agarstang** [comment]
@kirkins hadolint did not like having the `EXPOSE` as a variable at all (causes error not just failed test). I tried encapsulating it in a string and Docker did not like that.

**Comment by shrug** [comment]
Nice, thanks for the contribution! We're starting to try to get our arms around this repo, hopefully we'll have more regular updates moving forward

**Comment by underscorgan** [comment]
I don't think this use classifies for the caddy 'personal' license, I think we're going to need to move to a different webserver for the puppetexplorer image.

**Comment by underscorgan** [comment]
I don't think this use classifies for the caddy 'personal' license, I think we're going to need to move to a different webserver for the puppetexplorer image. I don't think this needs to change for this PR, more a general comment for @shrug for future

**Comment by agarstang** [comment]
@underscorgan I agree, this is why I brought it to attention. If the personal license doesn't cover dev then I'd remove it.

@e-gris what's the advantage to this? I try to consolidate my RUN commands to keep my layers neat.

**Comment by e-gris** [comment]
@agarstang shell scripts can be fussy and convoluted enough without having to deal with an additional semi-layer of interpretation as Docker interprets the RUN statement. 

Failures become harder to trace without an independent shell instance and the ever useful (I argue should be required '-e -u' flags)

Having the independent build script makes it possible to debug the container outside of Docker but still use the nearly the same method docker build uses. 

In short, it increases maintainability.

**Comment by agarstang** [comment]
@e-gris I agree, all valid points. I tend to be cautious when submitting pull requests to not change the "style" of the current code. I'm happy to make the modifications.

**Comment by e-gris** [comment]
@agarstang Understood. I left that more as generic commentary in case it might prove helpful.

File: puppetboard/Dockerfile [hunk 1]
```diff
@@ -23,15 +22,15 @@ RUN pip install puppetboard=="$PUPPET_BOARD_VERSION" gunicorn=="$GUNICORN_VERSIO
 
 COPY wsgi.py settings.py /var/www/puppetboard/
 
-EXPOSE $PUPPETBOARD_PORT
+EXPOSE 8000
```
Comment (e-gris) [inline puppetboard/Dockerfile]: What's the rationale behind moving this to a hard-code?

Comment (underscorgan) [reply puppetboard/Dockerfile]: from above, "hadolint did not like having the EXPOSE as a variable at all (causes error not just failed test). I tried encapsulating it in a string and Docker did not like that."
